com.unity.robotics.urdf-importer/Runtime/Math/Matrix3x3.cs

```diff
@@ -350,5 +350,10 @@ private bool IsTwoEigenvaluesEqual(float[] eigenvalues)
             return (eigenvalues[0] == eigenvalues[1] || eigenvalues[1] == eigenvalues[2] || eigenvalues[0] == eigenvalues[2]);
         }
 
+        public override string ToString()
+        {
+            return $"[[{elements[0][0]} {elements[0][1]} {elements[0][2]}]\n[{elements[1][0]} {elements[1][1]} {elements[1][2]}]\n[{elements[2][0]} {elements[2][1]} {elements[2][2]}]";
+        }
+
```

Comment:
Is this method used by any scripts?

Reply:
i added this for the purpose of the tests, so that in case of a failed test case, it'll print the expected/actual values instead of just the type

Reply:
Get you. Shall we make it an internal (or protected) method instead of public then?

Reply:
i get a `'Matrix3x3.ToString()': cannot change access modifiers when overriding 'public' inherited member 'object.ToString()'` if i change the access!

Reply:
Ah, okay; make sense